src/com/company/Home.java

```diff
@@ -0,0 +1,70 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Home {
+
+    private static Home INSTANCE;
+
+    private Scanner scanner;
+
+
+    public void init() {
+        scanner = new Scanner(System.in);
+        startMenu();
+
+    }
+
+    private void startMenu() {
+        System.out.println("Welcome to the Video Game Library.");
+            System.out.println(
+                            "1: Add a video game to your Library\n" +
+                            "2: Remove video game from your library\n" +
+                            "3: View the Library of your video games\n" +
+                            "4: Check out a game\n" +
+                            "5: View the checked out games\n" +
+                            "6: Check in a game\n" +
+                            "7: Close the program"
+            );
+            switch (scanner.next()) {
```

Comment:
All the actions should ideally be done directly on the library class. Is there any reason why you have separate classes to handle all the different actions that can be done to a game?